### PR TITLE
style: fix labels vertical align

### PR DESCRIPTION
**Pull request by mistercrunch**
### SUMMARY
recently merged PR https://github.com/apache/incubator-superset/pull/11400/ messed up the vertical alignment of `<Label />` in some contexts.

### BEFORE
Screen Shot 2020-10-23 at 9 49 50 PM
Screen Shot 2020-10-23 at 9 49 45 PM

### TEST PLAN
visual

### CODE REVIEW

**Comment by ktmud** [comment]
Just for the record, this PR also fixes the alignment of the datasource menu trigger on Explore page.

### Before



(misaligned by #11400 )

### After



### BTW...

Can we also add some space between the pill and the database name?



This label on the Edit Datasource modal also doesn't seem to need some adjustment

**Comment by mistercrunch** [comment]
How about this?
Screen Shot 2020-10-24 at 11 16 23 AM
Screen Shot 2020-10-24 at 11 16 12 AM
Screen Shot 2020-10-24 at 11 16 08 AM
Screen Shot 2020-10-24 at 11 16 03 AM

**Comment by ktmud** [comment]
I think text in brackets looks too hacky as it reminds me of text-based UI instead of GUI. Maybe we can create a special label with less padding for this?

**Comment by mistercrunch** [comment]
Ok, I'm throwing in the space for now but will save the `Label` work for another PR.

File: superset-frontend/src/components/Timer.tsx [hunk 1]
```diff
@@ -30,8 +30,8 @@ interface TimerProps {
 }
 
 const TimerLabel = styled(Label)`
-  width: 80px;
-  text-align: right;
+  width: 96px;
```
Comment (rusackas) [inline superset-frontend/src/components/Timer.tsx:33]: If this used the fixed-width / tabular-numerals variant if Inter, we could probably just as easily specify the width in  'em' units, but this seems like it'd work just fine for now, and I'll try to remember to open a PR to address these tabular numerals in the Theme.

**Review by rusackas — APPROVED** [review]

File: superset-frontend/src/components/Timer.tsx (hunk 1 above)
Comment (mistercrunch) [reply superset-frontend/src/components/Timer.tsx:33]: oh right, `em` seems like the way to go here!

**Pull request merged**